Public/Src/Tools/ContentPlacementAnalysisTools/ContentPlacementAnalysisTools.Extraction/ContentPlacementAnalysisTools.Extraction.dsc

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Tools.ContentPlacement.Extraction {
+
+    export declare const qualifier: BuildXLSdk.FullFrameworkQualifier;
+
+    // this is the log file config for cptools.builddownloader.exe
+    @@public
+    export const buildDownloaderLogFile = f`cptools.builddownloader.exe.nlog`;
+
+    // these are the static resources i use
+    @@public
+    export const staticResources = {
+        subfolder: "CPResources", contents : [{ 
+            subfolder : "Query", contents : [
+                f`CPResources\Query\get_build_data.kql`

```

Comment:
Since you just have one file, this is totally fine, but if this becomes more files, perhaps our helper to create a deployment from a directory would be easier:

```
export const staticResources = {
   subfolder: "CPResources, 
   contents: [
        importFrom("Sdk.Deployment").createFromDisk(d`CPResources`)
   ]
}
```